uefi-test-runner/src/proto/device_path.rs

```diff
@@ -48,4 +48,16 @@ pub fn test(image: Handle, bt: &BootServices) {
             .expect("Failed to convert text to device path");
         assert_eq!(path, convert);
     }
+
+    // Get the `LoadedImageDevicePath`. Verify it start with the same nodes as
+    // `device_path`.
+    let loaded_image_device_path = bt
+        .open_protocol_exclusive::<LoadedImageDevicePath>(image)
+        .expect("Failed to open LoadedImageDevicePath protocol");
+    for (n1, n2) in device_path
+        .node_iter()
+        .zip(loaded_image_device_path.node_iter())
+    {
+        assert_eq!(n1, n2);
```

Comment:
This only fails if the UEFI implementation is doing weird stuff, right?

Reply:
Right, I think this should always succeed. And per https://github.com/rust-osdev/uefi-rs/issues/553, the test is really only for OVMF anyway.